hack/test-shellspec.sh

```diff
@@ -29,7 +29,7 @@ for CHANGED in "${CHANGED_FILES[@]}"; do
             SPEC_DIRS=("${SPEC_DIRS[@]/${SPEC_DIR}}")
             [[ -n "${GITHUB_ACTIONS:-}" ]] && echo "::group::Shellspec test in ${SPEC_DIR}"
             echo -e "Detected changes in \033[1m${CHANGED_DIR}\033[0m, running tests"
-            PARAMS=(--chdir "${SPEC_DIR}" --shell /usr/bin/bash)
+            PARAMS=(--chdir "${SPEC_DIR}" --shell `which bash`)
```

Comment:
Seems unrelated to the changes?

Reply:
It is, but I left it here because it allows the script to run on a Mac, which I needed to test the change.

Reply:
where is bash on a mac??

Reply:
Still I'd probably move this up to line 22 or so and maybe comment it out:

```
BASH=/usr/bin/bash
# In case your bash is somewhere else
# BASH=$( which bash )

Reply:
What's the benefit of hardcoding the path?

Reply:
> where is bash on a mac??

If you're using homebrew it's at `/opt/homebrew/bin/bash`